.circleci/config.yml

```diff
@@ -105,6 +108,28 @@ jobs:
           root: ~/repo
           paths: .
 
+  react_test:
+    <<: *backstop_defaults
+    steps:
+      - attach_workspace: {at: /home/circleci}
+      - setup_remote_docker
+      - run: |
+          cd react
+          docker-compose up --no-start web backstop
+          docker cp ./ $(docker-compose ps -q backstop):/src/
+          docker cp ./storybook-static/. $(docker-compose ps -q web):/usr/share/nginx/html
+      - run: |
+          cd react
+          docker-compose run backstop test --atoms && docker-compose run backstop test
```

Comment:
I agree simplifying this is probably desired & I will test the speed differences in the next round of updates for this. For now I think we're going to add them as-is so we can unblock work on sharing SCSS a bit.

I set it up in a single run step so that if --atoms fails, that is the report that will be written, the second test config will not be run.

Again, agreed the complexity is probably not worth the speed savings overall & will probably be cleaned up in the next update.